### ISSUE TITLE
Feature request: Invite only players from other zones

### ISSUE THREAD

**Issue by BannZay** [iraizo/AutoLayer#30]
Sometimes you might not want to invite more players to your current layer since it possibly increases competition over npcs you are currently farming. Atm, the only way to stop inviting more players to your current zone is to disable invite feature completely (by left clicking on the minimap).

As an alternative, I suggest to add a feature that would let you ivnite only those players who is not currently in your zone.

Currently map icon can be
1.Green - invites turned on
2. Red - invites turned off

I am suggesting to add one more state
3. Yellow - invites turned on only for players from the zones other than your current one.

**Comment by iraizo** [comment]
I dont think this can be implemented properly due to the rate limit of /who requests to figure out where the player is

**Comment by BannZay** [comment]
Thats right. Thought, limited amount of invites (by /who qota) is still better than none.
